src/internal/packager/injector.go

```diff
@@ -177,8 +177,8 @@ func createPayloadConfigmaps(tempPath tempPaths, spinner *message.Spinner) ([]st
 		// Add the configmap to the configmaps slice for later usage in the pod
 		configMaps = append(configMaps, fileName)
 
-		// Give the control plane a slight buffeer
-		time.Sleep(100 * time.Millisecond)
+		// Give the control plane a 250ms buffer between each configmap
+		time.Sleep(250 * time.Millisecond)
```

Comment:
Yeah this is sort of arbitrary, there's no good way to make it not.  We're really just trying to not burden the control plane as much as it will produce event errors for users if they are watching them, though they are safe to ignore as well.